.github/workflows/dailyDLSmokeTest.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           results_history: acceptance-tests/results-history
 
       - name: Publish Github Pages
-        if: always()
```

Comment:
Do we need a gradle update here as well?

Reply:
@gds-jbentham yes, haven't got round to updating the PR here - fyi just noticed the dockerfile is also built off 8.1.1 version of gradle, shall I also update that/ is there a command I should run to build the dockerfile?